src/main/java/hu/oe/nik/szfmv/automatedcar/visualization/Dashboard.java

```diff
@@ -8,25 +18,75 @@
  */
 public class Dashboard extends JPanel {
 
-    private final int width = 250;
-    private final int height = 700;
-    private final int backgroundColor = 0x888888;
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    private static final int WIDTH = 250;
+    private static final int HEIGHT = 700;
+    private static final int BACKGROUND_COLOR = 0x888888;
+
+    private Gui parent;
+
+    private TurnSignal leftTurnSignal;
+    private TurnSignal rightTurnSignal;
+    private TextSignal accSpeedSignal;
+    private TextSignal accDistanceSignal;
+    private TextSignal accOnOffSignal;
+    private TextSignal ppSignal;
+    private TextSignal lkaSignal;
+    private TextSignal lkaWarningSignal;
+    private TextSignal trafficSignSignal;
+    private TextSignal aebWarningSignal;
+    private TextSignal rrWarningSignal;
+    private DashboardText currentGear;
+    private DashboardText speedLimit;
+    private DashboardText steeringWheel;
+    private DashboardText xCoordinate;
+    private DashboardText yCoordinate;
+    private ProgressBars gasProgressBar;
+    private ProgressBars breakProgressBar;
+    private Gauge rpmGauge;
+    private Gauge kmhGauge;
 
-    Gui parent;
 
     private Thread timer = new Thread() {
-        int difference;
+        //int difference;    // ez nem tudom mire valo, egyelore maradjon
 
         public void run() {
             while (true) {
-
                 try {
-                    Thread.sleep(100);
+                    VirtualFunctionBus vfb = parent.getVirtualFunctionBus();
+                    if (vfb != null) {
+                        handleInputPacket(vfb.inputPacket);
+
+                        //...
+                    }
+
+                    Thread.sleep(40);
                 } catch (InterruptedException ex) {
+                    LOGGER.error("Error in Dashboard timer thread!", ex);
                 }
-
             }
         }
+
+        private void handleInputPacket(ReadOnlyInputPacket inputPacket) {
+            //rpmGauge.setValue(3000);      Read from the bus
+            //kmhGauge.setValue(130);       Read from the bus
```

Comment:
Ezek megint mire föl vannak itt?
Ez volt a buszról érkező adat mock-olása?
Akkor kéne egy mock kapcsoló, ha mock-olás van, akkor ezek kerülnek beállításra egyébként meg a buszról olvasás jön...

Ez itt így szemét.

Reply:
Ugyanaz mint az előző. Annyi hogy ezek nem is itt, hanem egy másik handle...Packet metódusban lesznek. Ez csak egy jelzés hogy ezeket kell még rendesen beállítanunk, kb. mint egy TODO. Csak a TODO-kon elhasalt a codefactor a pull request check-eknél, ezért nem használjuk.
Mondjuk kitehettük volna ezeket egy külön handle...Packet()-be, és ott beállítjuk a konstans értékeket amíg nem integrálunk a másik csapattal. Csináljuk ezt? És ha igen akkor ehhez is új branch-et kell használnunk?

Reply:
A külön packet kifejezetten jól hangzik. 
Illene, de most az idő rövidsége miatt hagyjuk.